### PR TITLE
Grammatically updated the docs.

### PR DESCRIPTION
**Pull request by KevalPrajapati**
A small change to a line on [docs](https://flutter.dev/docs#docs) which would make the sentence complete.

### CODE REVIEW

**Comment by KevalPrajapati** [comment]
@googlebot I signed it!

**Review by sfshaza2 — CHANGES_REQUESTED** [review]
This is an ok edit, though "Flutter" needs to be capitalized. :)

**Comment by KevalPrajapati** [comment]
> This is an ok edit, though "Flutter" needs to be capitalized. :)

Okay. I would do that.

**Comment by KevalPrajapati** [comment]
> This is an ok edit, though "Flutter" needs to be capitalized. :)

Done the changes.

**Comment by KevalPrajapati** [comment]
> This is an ok edit, though "Flutter" needs to be capitalized. :)

Done the changes in the second commit. Please review.

File: src/docs/index.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ here are some next steps.
 
 ### Docs
 
-Coming from another platform? Check out flutter for:
+Coming from another platform? Check out Flutter for:
```
Comment (KevalPrajapati) [inline src/docs/index.md:38]: Capitalized F in Flutter.

**Pull request closed without merging**